### PR TITLE
Person b record endpoints

### PR DESCRIPTION
**Pull request by Laszlo-JFLMTCO**
@akintner 
This Pull Request covers all serializer additions as well as show endpoints for the followings:  
  * invoice  
  * item  
  * invoice item
It has also the branch from PR #16 and closes #17 and #10 .
FYI @case-eee

### CODE REVIEW

**Comment by akintner** [comment]
looks good, ready for merge to master

File: app/serializers/invoice_item_serializer.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+class InvoiceItemSerializer < ActiveModel::Serializer
+  attributes :id, :item_id, :invoice_id, :quantity, :unit_price, :created_at, :updated_at
```
Comment (case-eee) [inline app/serializers/invoice_item_serializer.rb:2]: @akintner i like that you're using serializers 👍 is this any different from the data in the db though?

**Review by case-eee — COMMENTED** [review]
👍

File: app/serializers/invoice_item_serializer.rb (hunk 1 above)
Comment (akintner) [reply app/serializers/invoice_item_serializer.rb:2]: @case-eee  Yes, this is the data from the DB. We were under the impression that the models themselves would come through with all their data, but any model associations on other models would just come through with IDs. That seemed to be what the spec for the project was saying, at least as we interpreted it.

**Pull request merged**